deltachat-ios/Controller/ChatListViewController.swift

```diff
@@ -134,9 +134,14 @@ class ChatListViewController: UITableViewController {
 
         // update messages - for new messages, do not reuse or modify strings but create new ones.
         // it is not needed to keep all past update messages, however, when deleted, also the strings should be deleted.
-        let msg = dcContext.newMessage(viewType: DC_MSG_TEXT)
-        msg.text = String.localized("update_1_42_common") + "\n\n" + String.localized("more_info_desktop") +  ": https://get.delta.chat/#changelogs"
-        dcContext.addDeviceMessage(label: "update_1_42b_ios", msg: msg)
+        let deviceMsgLabel = "update_1_42b_ios2"
+        if !dcAccounts.isJustAdded(id: dcContext.id) {
+            let msg = dcContext.newMessage(viewType: DC_MSG_TEXT)
+            msg.text = String.localized("update_1_42_common") + "\n\n" + String.localized("more_info_desktop") +  ": https://get.delta.chat/#changelogs"
+            dcContext.addDeviceMessage(label: deviceMsgLabel, msg: msg)
+        } else {
+            dcContext.addDeviceMessage(label: deviceMsgLabel, msg: nil)
```

Comment:
Why is adding a `nil`-message necessary?

Reply:
a nil-message is added to mark the label as used; this avoids the message being added on the next start.

[`dc_add_device_msg()`](https://c.delta.chat/classdc__context__t.html#a1a2aad98bd23c1d21ee42374e241f389) tries to explain that maybe better :)